File: tests/c_programs/multiple_paths.c
int classify(int x) {
    if (x < 0) {
        return -1;
    } else if (x == 0) {
        return 0;
    } else if (x < 10) {
        return 1;
    } else if (x < 100) {
        return 2;
    } else {
        return 3;
    }
}

File: tests/rust_programs/multiple_paths.rs
fn classify(x: i32) -> i32 {
    if x < 0 {
        -1
    } else if x == 0 {
        0
    } else if x < 10 {
        1
    } else if x < 100 {
        2
    } else {
        3
    }
}